fern/styles/performance-dashboard.css

```
@@ -0,0 +1,307 @@
+.performance-dashboard {
+  --pd-nv-green: #76b900;
+  --pd-intel-blue: #0071c5;
+  --pd-border: #e0e0e0;
+  --pd-muted: #666;
+  margin: 0 0 24px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)